R/fancy_ideogram.R

```diff
@@ -77,9 +77,9 @@ fancy_ideogram = function(this_sample_id,
   }
 
   #grch37 coordinates
-  grch37_end = GAMBLR.data::chromosome_arms_grch37[c(2,4,6,8,10,12,14,16,18,20,22,24,26,28,30,32,34,36,38,40,42,44),3]
-  grch37_cent_start = GAMBLR.data::chromosome_arms_grch37[c(1,3,5,7,9,11,13,15,17,19,21,23,25,27,29,31,33,35,37,39,41,43),3]
-  grch37_cent_end = GAMBLR.data::chromosome_arms_grch37[c(2,4,6,8,10,12,14,16,18,20,22,24,26,28,30,32,34,36,38,40,42,44),2]
+  grch37_end = GAMBLR.data::chromosome_arms_grch37[c(2,4,6,8,10,12,14,16,18,20,22,24,26,28,30,32,34,36,38,40,42,44,46,48),3]
```

Comment:
I agree, this process could likely be updated and make use of more efficient data wrangling with dplyr to achieve the same thing.

Reply:
So is this going to be fixed? Alternatively, can this remain as is and the sex chromosomes are just ignored? Either way, fixed or not, the rest of the function will have to be updated accordingly

Reply:
Should chromosome X be ignored as well?

Reply:
Sure you can include X if you want but definitely exclude Y because it is always ignored since without knowing the sex it is impossible to infer it or have accurate calls for samples without matched normal.

Reply:
- The code chunk was changed to be flexible and not hard-coded.
- Y chromosome was ignored.